lib/libc/syslog/lib_lowsyslog.c

```diff
@@ -150,9 +150,7 @@ int lowvsyslog(int priority, FAR const char *fmt, va_list ap)
 #if defined(CONFIG_LOGM) && defined(CONFIG_SYSLOG2LOGM)
 		ret = logm_internal(LOGM_LOWPUT, LOGM_UNKNOWN, priority, fmt, ap);
 #else
-		irqstate_t flags = irqsave();
```

Comment:
As far as I know, we added this to prevent log mix. Don't we need this anymore?

Reply:
@abhishek-samsung Can you pls check and reply on this?

Reply:
Please revert the change. I have added it only for debugging purpose.

Reply:
Done